datajunction-server/datajunction_server/api/system.py

```diff
@@ -0,0 +1,141 @@
+"""
+Router for various system overview metrics
+"""
+
+import logging
+import hashlib
+from fastapi import BackgroundTasks, Depends, Query
+from sqlalchemy import select, text
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from datajunction_server.internal.access.authorization import (
+    validate_access,
+)
+from datajunction_server.models import access
+from datajunction_server.models.system import DimensionStats, RowOutput
+from datajunction_server.sql.dag import (
+    get_cubes_using_dimensions,
+    get_dimension_dag_indegree,
+)
+from datajunction_server.api.sql import get_node_sql
+from datajunction_server.internal.caching.cachelib_cache import get_cache
+from datajunction_server.internal.caching.interface import Cache
+from datajunction_server.database.user import User
+from datajunction_server.database.node import Node
+from datajunction_server.internal.access.authentication.http import SecureAPIRouter
+from datajunction_server.models.node_type import NodeType
+from datajunction_server.utils import (
+    get_current_user,
+    get_session,
+    get_settings,
+)
+
+logger = logging.getLogger(__name__)
+router = SecureAPIRouter(tags=["System"])
+
+
+@router.get("/system/metrics")
+async def list_system_metrics(
+    session: AsyncSession = Depends(get_session),
+):
+    """
+    Returns a list of DJ system metrics (available as metric nodes in DJ).
+    """
+    settings = get_settings()
+    metrics = await Node.find_by(
+        session=session,
+        namespace=settings.seed_setup.system_namespace,
+        node_types=[NodeType.METRIC],
+    )
+    return [m.name for m in metrics]
+
+
+@router.get("/system/data/{metric_name}")
+async def get_data_for_system_metrics(
```

Comment:
This function is getting data for one system metric:
```suggestion
async def get_data_for_system_metric(
```